src/components/TextInput/TextInput.js

```diff
@@ -41,6 +46,9 @@ export default class TextInput extends Component {
     invalid: false,
     labelText: '',
     invalidText: 'Provide invalidText',
+    clearable: false,
+    clearDescription: 'Clear',
```

Comment:
i'd put something here to the effect of `Provide 'clear' text` to be clear that this prop needs to be passed in if `clearable` is true